gnovm/tests/imports.go

```diff
@@ -567,6 +567,53 @@ func testPackageInjector(store gno.Store, pn *gno.PackageNode) {
 				m.Context = ctx
 			},
 		)
+		pn.DefineNative("TestSetPrevRealm",
+			gno.Flds( // params
+				"", gno.AnyT(),
```

Comment:
Why not have this be a string input instead of any?

Reply:
for parameter type, both `std.Address` and `string` can be used.
- if passed type is `string`, it will set PrevRealm to a `Realm.
- if passed type is `std.Address`, it will set PrevRealm to a (normal) user

if `TestSetPrevRealm` takes only string type, we have to determine whether that value is user address or pkg_path, which we need clear way to determine it.

Reply:
I would prefer to have specific methods for specific cases, like `TestSetPRevRealm` and `TestSetPrevUser`, instead of using `any` making it more difficult for the user to figure out what that method accepts.

Reply:
> 

good point, I really haven't thought about dividing these kind of functions. I'll divide it